frame/examples/offchain-worker-price-oracle/src/lib.rs

```diff
@@ -0,0 +1,529 @@
+	#[pallet::getter(fn prices)]
+	pub(super) type Prices<T: Config> = StorageValue<_, BoundedVec<u32, T::MaxPrices>, ValueQuery>;
+
+	#[pallet::storage]
+	#[pallet::getter(fn authorities)]
+	pub(super) type Authorities<T: Config> =
+		StorageValue<_, BoundedVec<T::AccountId, T::MaxAuthorities>, ValueQuery>;
+
+	#[pallet::error]
+	pub enum Error<T> {
+		NotAuthority,
+		AlreadyAuthority,
+		TooManyAuthorities,
+	}
+
+	#[pallet::pallet]
+	pub struct Pallet<T>(_);
+
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		/// Offchain Worker entry point.
+		///
+		/// By implementing `fn offchain_worker` you declare a new offchain
+		/// worker. This function will be called when the node is fully synced
+		/// and a new best block is successfully imported.
+		///
+		/// Note that it's not guaranteed for offchain workers to run on EVERY
+		/// block, there might be cases where some blocks are skipped, or for
+		/// some the worker runs twice (re-orgs), so the code should be able to
+		/// handle that.
+		///
+		/// You can use `Local Storage` API to coordinate runs of the worker.
+		fn offchain_worker(block_number: BlockNumberFor<T>) {
+			// Note that having logs compiled to WASM may cause the size of the
+			// blob to increase significantly. You can use `RuntimeDebug` custom
+			// derive to hide details of the types in WASM. The `sp-api` crate
+			// also provides a feature `disable-logging` to disable all logging
+			// and thus, remove any logging from the WASM.
+			log::info!("Hello World from offchain workers!");
+
+			// Since off-chain workers are just part of the runtime code, they
+			// have direct access to the storage and other included pallets.
+			//
+			// We can easily import `frame_system` and retrieve a block hash of
+			// the parent block.
+			let parent_hash = <system::Pallet<T>>::block_hash(block_number - 1u32.into());
+			log::debug!("Current block: {:?} (parent hash: {:?})", block_number, parent_hash);
+
+			// It's a good practice to keep `fn offchain_worker()` function
+			// minimal, and move most of the code to separate `impl` block. Here
+			// we call a helper function to calculate current average price.
+			// This function reads storage entries of the current state.
+			let average: Option<u32> = Self::average_price();
+			log::debug!("Current price: {:?}", average);
+
+			/// A friendlier name for the error that is going to be returned in
+			/// case we are in the grace period.
+			const RECENTLY_SENT: () = ();
+
+			// Start off by creating a reference to Local Storage value. Since
+			// the local storage is common for all offchain workers, it's a good
+			// practice to prepend your entry with the module name.
+			let val = StorageValueRef::persistent(b"example_ocw::last_send");
+			// The Local Storage is persisted and shared between runs of the
+			// offchain workers, and offchain workers may run concurrently. We
+			// can use the `mutate` function, to write a storage entry in an
```

Comment:
The whole line wrapping here seems 80 rather than 100 🙄

Reply:
@kianenigma it seems that your comment is pointing to the code before [this commit](https://github.com/paritytech/substrate/pull/14280/commits/f96dacdf4c980349d7dee37d5be9c9c133e02538), which addresses line wrapping 

here's the same snippet after the commit:
https://github.com/paritytech/substrate/blob/5ec0432f479fb1c5b49476e439642923cb0c20d7/frame/examples/offchain-worker-price-oracle/src/lib.rs#L230-L236